### PR TITLE
Add "modules at a glance" table

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
### Purpose/implementation Section

I personally merge many of the pull requests that come through this repository. As a result, I have a working knowledge of what modules consume which files and how modules are interdependent/which modules create files consumed by other analyses. I am documenting that knowledge via a table that I'm terming the "modules at a glance" table, which I am adding to the README for `analyses` here. It is intended to make it easier for other project organizers and reviewers to get quickly get an overview of all of the modules.

It is designed to note a module's name, what files it uses as input, a high-level description of what the module does and its status, and what output files it produces that other modules rely on. I think this is going to be easier to maintain than some kind of chart.

#### What GitHub issue does your pull request address?

Related to the "what data analyses consume" part of https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/336#issuecomment-565624542

#### Is there anything that you want to discuss further?

Will this modules at a glance table reduce the burden for other project organizers? How can it be improved? (Also did I successfully list all the `analyses` directories? 😬 )

### CODE REVIEW

File: analyses/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+| [`fusion_filtering`](https://github.com/jaclyn-taroni/OpenPBTA-analysis/tree/analyses-readme/analyses/fusion_filtering) | `pbta-fusion-arriba.tsv.gz`, `pbta-fusion-starfusion.tsv.gz` | Standardizes, filters, and prioritizes fusion calls | `PutativeDriverFusion.tsv` which is distributed as `pbta-fusion-putative-oncogenic.tsv` in the data download 
+| [`independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | `pbta-histologies.tsv` | Generates independent specimen lists for WGS/WXS samples | `independent-specimens.wgs.primary.tsv`, `independent-specimens.wgs.primary-plus.tsv`, `independent-specimens.wgswxs.primary.tsv`, `independent-specimens.wgswxs.primary-plus.tsv` (included in data download)
+| [`interaction-plots`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/interaction-plots) | Current version uses `pbta-snv-lancet.vep.maf.gz` | Creates interaction plots for mutation mutual exclusivity/co-occurrence [#13](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/13); may be updated to include other data types (e.g., fusions) | N/A
```
Comment (jashapiro) [inline analyses/README.md]:
```suggestion
| [`interaction-plots`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/interaction-plots) | `independent-specimens.*.tsv`, Current version uses `pbta-snv-lancet.vep.maf.gz` | Creates interaction plots for mutation mutual exclusivity/co-occurrence [#13](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/13); may be updated to include other data types (e.g., fusions) | N/A
```

**Comment by jaclyn-taroni** [comment]
What do you think of the wording in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/345/commits/43489e51c335b4b0759cb3c4d54ab626dba7b838 @jashapiro ? If we do that should we add something to the main README instructions as well?

**Comment by jashapiro** [comment]
> What do you think of the wording in [43489e5](https://github.com/AlexsLemonade/OpenPBTA-analysis/commit/43489e51c335b4b0759cb3c4d54ab626dba7b838) @jashapiro ? If we do that should we add something to the main README instructions as well?

I suppose we should.

File: README.md [hunk 2]
```diff
@@ -175,6 +175,17 @@ OpenPBTA-analysis
...
 ```
 
+### Documenting Your Analysis
+
+A goal of the OpenPBTA project is to create a collection of workflows that are commonly used for atlas papers.
+As such, documenting your analytical code via comments and including information summarizing the purpose of your analysis is important.
+
+When you file the first pull request creating a new analysis module, add your module to the [Modules At A Glance table](analyses#modules-at-a-glance).
+This table contains fields for the directory name, what input files are required, a short description, and any files that you expect other analyses will rely on.
+This item is included in the pull request reproducibility checklist.
```
Comment (jashapiro) [inline README.md]:
```suggestion
As your analysis develops and input and output files change, please check this table remains up to date. 
This step is included in the pull request reproducibility checklist.
```

**Pull request merged**